specification/search/data-plane/Microsoft.Azure.Search.Service/preview/2017-11-11-preview/examples/SearchServiceCreateIndex.json

```diff
@@ -41,8 +41,8 @@
                 },
                 {
                     "name": "tags",
-                    "type": "Collection(Edm.String)",
-                    "analyzer": "tagsAnalyzer"
```

Comment:
@brjohnstmsft While we are discussing about Collection(Edm.String), Could you explain the reason behind the tagsAnalyzer? Is that correct value? Or is it Ok to change that?

Reply:
This is another instance of the same problem. In Azure Search, an analyzer is a named configuration for processing text that is indexed by or sent as a query to the search engine. There are 50+ built-in analyzers for various languages (e.g. -- 'microsoft.en' is the English analyzer provided by Microsoft), but customers can also define their own custom analyzers and given them their own names. `tagsAnalyzer` is the name of such a custom analyzer. Since this is extensible by the customer, we wish to model it as an extensible enum, just like `DataType`, `RegexFlags`, etc.

Reply:
@brjohnstmsft Thanks for the clarification. I will resolve this conversation now.